site/pages/react/authentication-methods.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-title: Authenticate Users
+title: Authentication Methods
 description: Learn how to authenticate users using Account Kit in a React application.
 ---
 
-# Authenticate users
+# Authentication methods
 
 Account Kit makes it really easy to authenticate your users using a number of different authentication methods. If you followed the [quickstart](/react/quickstart), then your app is already setup to authenticate users and you can skip this guide!
```

Comment:
Let's add that a person can "choose to do either" 1 or 2 below.

Reply:
specified this by using the phrase "we'll show you two different ways to authenticate users"